Tensile/Source/client/source/Reference.cpp

```diff
@@ -411,126 +411,118 @@ namespace Tensile
                 std::cout << " " << counts.description() << "\n";
             }
 
-            // Loops always traverse in same order but addressing in memory can be
-            // flexible to support different activation and filter formats
-            std::vector<size_t> s(ConvolutionProblem::MaxNumSpatialDims, 0);
-            std::vector<size_t> f(ConvolutionProblem::MaxNumSpatialDims, 0);
+            // Loops always traverse in same order but addressing in memory can be flexible to support different activation
+            // and filter formats
+            size_t spatialCoordCount = CoordCount(counts.scount.begin(), counts.scount.end());
+#pragma omp parallel for collapse(3)
             for(size_t cout = 0; cout < counts.coutCount; cout++)
```

Comment:
I would hesitate to use 'cout' as a variable name, for the most obvious reason being if anyone using std::cout would find this out in a hurry.